internal/kms/vault_sa.go

```diff
@@ -35,6 +36,9 @@ const (
 	// should be available in the Tenants namespace. This ServiceAccount
 	// will be used to connect to Hashicorp Vault.
 	vaultTenantSAName = "ceph-csi-vault-sa"
+	// Kubernetes version to get token from the ServiceAccount.
+	kubeMajorVersion = "1"
```

Comment:
the name of these constants do not say anything about the feature that Kubernetes introduced. Please use a better name.